php-classes/Slate/Courses/SectionsRequestHandler.php

```diff
@@ -208,6 +208,15 @@ public static function handleBrowseRequest($options = [], $conditions = [], $res
             $conditions[] = sprintf('ID IN (%s)', count($enrolledSectionIds) ? join(',', $enrolledSectionIds) : '0');
         }
 
+        if (!empty($_REQUEST['sort_current'])) {
+            try {
+                $termIds = Term::getCurrent()->getMaster()->getContainedTermIDs();
+                if (!empty($termIds)) {
+                    $options['order'] = sprintf('FIELD(`%s`.TermID, %s) DESC', Section::getTableAlias(), join(", ", $termIds));
+                }
+            } catch (\Exception $e) {}
+        }
```

Comment:
@nbey I think we discussed this somewhere else, this should be implemented using a `$sorters` config on the ActiveRecord which can then be controlled by the `sort` parameter implemented generically by `RecordsRequestHandler`. We don't want separate competing sort parameters, new query params should be used only for filters